common/src/main/java/com/box/l10n/mojito/po/PoPluralRule.java

```diff
@@ -97,6 +101,8 @@
         mappingForNonDefault.put("sk", PoPluralRule.THREE_FORMS_SPECIAL_FOR_1_2_3_4);
         mappingForNonDefault.put("pl", PoPluralRule.THREE_FORMS_SPECIAL_FOR_1_AND_ENDING_234);
         mappingForNonDefault.put("sl", PoPluralRule.THREE_FORMS_SPECIAL_ENDING_020310_AND_1199); // incompatible with CLDR?
+        mappingForNonDefault.put("he", PoPluralRule.FOUR_FORMS_FRACTIONAL_DIGITS_OTHER);
+        mappingForNonDefault.put("iw", PoPluralRule.FOUR_FORMS_FRACTIONAL_DIGITS_OTHER);
```

Comment:
Added "iw" as locales containing regions e.g. 'he-IL', get converted to 'iw-IL' code which then uses the default plural rules unless it's handled in this map